plugins/french/novhell.ts

```diff
@@ -0,0 +1,209 @@
+import { CheerioAPI, load } from 'cheerio';
+import { fetchApi, fetchFile } from '@libs/fetch';
+import { Plugin } from '@typings/plugin';
+import { Filters, FilterTypes } from '@libs/filterInputs';
+import { defaultCover } from '@libs/defaultCover';
+import { NovelStatus } from '@libs/novelStatus';
+
+class NovhellPlugin implements Plugin.PluginBase {
+  id = 'novhell';
+  name = 'Novhell';
+  icon = 'src/fr/novhell/icon.png';
+  site = 'https://novhell.org';
+  version = '1.0.0';
+  filters: Filters | undefined = undefined;
+
+  async getCheerio(url: string): Promise<CheerioAPI> {
+    const r = await fetchApi(url, {
+      headers: { 'Accept-Encoding': 'deflate' },
+    });
+    const body = await r.text();
+    const $ = load(body);
+    return $;
+  }
+
+  async popularNovels(
+    pageNo: number,
+    {
+      showLatestNovels,
+      filters,
+    }: Plugin.PopularNovelsOptions<typeof this.filters>,
+  ): Promise<Plugin.NovelItem[]> {
+    if (pageNo > 1) return [];
+
+    const novels: Plugin.NovelItem[] = [];
+    let novel: Plugin.NovelItem;
+    let url = this.site;
+    let $ = await this.getCheerio(url);
+    $('article div div div figure').each((i, elem) => {
+      let novelName = $(elem)
+        .find('figcaption span strong')
+        .first()
+        .text()
+        .trim();
+      if (!novelName || novelName.trim() === '') {
+        novelName = $(elem).find('figcaption a strong').first().text().trim();
+      }
+      const novelCover = $(elem).find('a img').attr('src');
+      const novelUrl = $(elem).find('a').attr('href');
+
+      if (novelUrl && novelName) {
+        novel = {
+          name: novelName,
+          cover: novelCover,
+          path: novelUrl,
```

Comment:
novel path doesnt include the site domain
eg: `/novel/1` instead of `https://site.com/novel/1`